src/mongocxx/collection.cpp

```diff
@@ -65,6 +65,23 @@ stdx::string_view collection::name() const noexcept {
     return _impl->name;
 }
 
+void collection::name(stdx::string_view new_name, bool overwrite_old) {
+  bson_error_t error;
+
+  _impl->name = (std::string)new_name;
```

Comment:
understood. In that case, please use static_cast, we try to avoid C-style casts.